terraform/gcp/variables.tf

```diff
@@ -4,6 +4,12 @@ variable "bigquery-project-id" {
   default     = ""
 }
 
+variable "block-project-ssh-keys" {
+  type        = bool
+  description = "Block project-wide SSH keys from being able to connect to the instance"
```

Comment:
that means that you need to create specific keys to get in the instance?

Reply:
There's something called "OS Login" that sounds like it is the preferred method of logging in to instances in GCP. It seems the main difference is that SSH keys become linked to IAM profiles, so you can grant access to only the required individual instances rather than global access.

https://cloud.google.com/compute/docs/instances/access-overview

Reply:
I've updated the PR to also add the ability to enable OS Login on the instance, although it defaults to `false` (i.e. defaults to no instance access unless specified)